src/index.html

```diff
@@ -136,7 +136,7 @@
                   >#sponsors</a
```

Comment:
More gap please  
![Screenshot 2022-06-17 at 22-22-07 meet js Summit 2022](https://user-images.githubusercontent.com/509375/174396265-51d2f11b-2ea9-427f-ad74-4a5c15d5457e.png)

Reply:
While we're at it - unbreakable spaces please, so it's a single bar
![Screenshot 2022-06-17 at 22-23-23 meet js Summit 2022](https://user-images.githubusercontent.com/509375/174396505-42fae475-d9e9-470e-bb9c-a9343b1ff69e.png)